orwell/proxy/main.py

```diff
@@ -183,7 +224,7 @@ def send_input(self, data):
             index, _, joystick_type = info.partition(' ')
             index = int(index)
             self._joysticks[index] = orwell.proxy.input.Joystick(
-                0.2,
+                0.05,
```

Comment:
This will deserve its own pull request ; this will happen a bit later as there are a few more changes to convert to pull requests pending.